crates/fuel-core/Cargo.toml

```diff
@@ -44,6 +45,7 @@ hyper = { workspace = true }
 indicatif = { workspace = true, default-features = true }
 itertools = { workspace = true }
 num_cpus = { version = "1.16.0", optional = true }
+paste = "1"
```

Comment:
We use it in several places, I think it will be better to use `{ workspace = true }` isntead

Reply:
Sure. 1e70c771aa

Reply:
We have two more places=)

<img width="959" alt="image" src="https://github.com/user-attachments/assets/b86ead95-7b23-4528-ac08-256e8a0113c7">